data/upgrades/crew.json

```diff
@@ -1272,5 +1272,108 @@
     "cost": { "value": 9 },
     "restrictions": [{ "factions": ["Resistance"] }],
     "hyperspace": true
+  },
+  {
+    "name": "Chancellor Palpatine",
+    "limited": 1,
+    "xws": "chancellorpalpatine",
+    "sides": [
+      {
+        "title": "Chancellor Palpatine",
+        "type": "Crew",
+        "ability": "Setup: Equip this side faceup. After you defend, if the attacker is at range 0-2, you may spend 1 [Force Charge]. If you do, the attacker gains 1 stress token. During the End Phase, you may flip this card.",
+        "image": "https://images-cdn.fantasyflightgames.com/filer_public/85/67/85679b7f-d99c-4d8f-8f5c-abd0ffda7842/swz30_palpatine.png",
+        "force": {
+          "value": 1,
+          "recovers": 1
+        },
+        "grants": [
+          {
+            "type": "action",
+            "value": {
+              "type": "Coordinate",
+              "difficulty": "Purple"
+            },
+            "amount": 1
+          }
+        ],
+        "slots": ["Crew"]
+      },
+      {
+        "title": "Darth Sidious",
+        "type": "Crew",
+        "ability": "After you perfor a purple [Coordinate] action, the ship you coordinated gains 1 stress token. Then, it gains 1 focus token or recovers 1 [Force Charge].",
+        "image": "https://images-cdn.fantasyflightgames.com/filer_public/53/14/5314a1dc-e52f-4239-ad1e-3f9a3834e604/swz30_sidious.png",
+        "force": {
+          "value": 1,
+          "recovers": 1
+        },
+        "grants": [
+          {
+            "type": "action",
+            "value": {
+              "type": "Coordinate",
+              "difficulty": "Purple"
+            },
+            "amount": 1
+          }
+        ],
+        "slots": ["Crew"]
+      }
+    ],
+    "restrictions": [
+      {
+        "factions": ["Galactic Republic", "Separatist Alliance"]
+      }
+    ],
+    "hyperspace": false
+  },
+  {
+    "name": "Count Dooku",
+    "limited": 1,
+    "xws": "countdooku-crew",
```

Comment:
We haven't included the upgrade type suffix if it's the first upgrade with this name

Reply:
Oh, my bad, I misunderstood an error from the tests and tried adding that as fix and forgot to remove it when I found the problem (which was that I had accidentally duplicated Dooku).